$DragonFly: src/usr.sbin/ntpd/patches/Attic/server.c.patch,v 1.2 2005/02/28 21:40:49 joerg Exp $

--- server.c.orig	2004-11-09 20:34:55.000000000 +0100
+++ server.c	2004-11-09 20:38:09.000000000 +0100
@@ -0,0 +0,0 @@
